nornir_napalm/plugins/tasks/napalm_confirm_commit.py

```diff
@@ -0,0 +1,33 @@
+from typing import Optional
+from nornir.core.task import Result, Task
+
+from nornir_napalm.plugins.connections import CONNECTION_NAME
+
+
+def napalm_confirm_commit(task: Task, dry_run: Optional[bool] = None) -> Result:
```

Comment:
Just a minor suggestion (feel free to ignore). I think `dry_run: Optional[bool] = None` could be `dry_run: bool = False` simplifying slightly the signature and having no impact on the code.

Reply:
disregard, consistency with others...